dapps/assets/src/components/digitaltwins/AddDigitalTwin.ts

```diff
@@ -250,19 +245,21 @@ class AddDigitalTwinComponent extends mixins(EvanComponent) {
       });
     }
 
-    Object.keys(template.plugins).forEach((pluginKey: string) => {
-      const { description } = template.plugins[pluginKey];
-      const errors = this.runtime.description.validateDescription({
-        public: description,
-      });
-
-      if (Array.isArray(errors) && errors.length > 0) {
-        TemplateErrorInterfaces.push({
-          name: pluginKey,
-          errors,
+    if (template.plugins) {
```

Comment:
or early return:
```
if (!template.plugins) {
    return TemplateErrorInterfaces;
....
```

Reply:
Would result in double return. Is it better? :D

```
 if (!template.plugins) {
      return TemplateErrorInterfaces;
    }

    if (template.plugins) {
      Object.keys(template.plugins).forEach((pluginKey: string) => {
        const { description } = template.plugins[pluginKey];
        const errors = this.runtime.description.validateDescription({
          public: description,
        });

        if (Array.isArray(errors) && errors.length > 0) {
          TemplateErrorInterfaces.push({
            name: pluginKey,
            errors,
          });
        }
      });
    }

    return TemplateErrorInterfaces;
```

Reply:
nope, second if may be omitted than:
```javascript
if (!template.plugins) {
  return TemplateErrorInterfaces;
}

Object.keys(template.plugins).forEach((pluginKey: string) => {
  const { description } = template.plugins[pluginKey];
  const errors = this.runtime.description.validateDescription({
    public: description,
  });

  if (Array.isArray(errors) && errors.length > 0) {
    TemplateErrorInterfaces.push({
      name: pluginKey,
      errors,
    });
  }
});

return TemplateErrorInterfaces;
```

Reply:
kk, i will use it.